src/main/java/zd/zero/waifu/motivator/plugin/platform/UpdateAssetsListener.kt

```diff
@@ -0,0 +1,11 @@
+package zd.zero.waifu.motivator.plugin.platform
+
+import com.intellij.util.messages.Topic
+
+interface UpdateAssetsListener {
+    companion object {
+        val TOPIC = Topic.create("Motivator Assets Update", UpdateAssetsListener::class.java)
+    }
+
+    fun requestUpdate()
```

Comment:
*naming-discussion*: this method is subscribed in the message publisher and would it be okay if we used the convention `on*`? wdyt about `onRequestUpdate()`?

Reply:
Yeah I can put that back

I just saw this

```kotlin
        ApplicationManager.getApplication().messageBus
            .syncPublisher(UpdateAssetsListener.TOPIC)
            .onRequestUpdate()
```
and thought it looked weird

Reply:
Hmmm, for me it doesn't look that weird 😅, maybe because I'm not a native speaker as it translates to me like:
"on the event of request update" compared to a command-ish "Hey computer, request update from assets" (`requestUpdate`)

Reply:
Ah what was weird to me was that I was `publishing` an event by invoking `onRequestUpdate`.
I saw the act of publishing an event to the topic by using `onRequestUpdate` weird and preferred to publish an event using `requestUpdate` which dispatched the update event. However, I didn't care that much :smile: